content/opt/run

```diff
@@ -219,5 +212,5 @@ echo "==================================================================="
 touch ${initfile}
 fi
 
-echo "Starting Supervisor.  You can safely CTRL-C and the container will continue to run with or without the -d (daemon) option"
+echo "Starting Supervisor."
```

Comment:
I added this note  to let users know it was OK to break out as there seemed to be a lot of confusion for new users.  I think we should keep it in for now 😛